FordLabs/retroquest
# Adds Action Radiator View

## Overview
This PR adds in an action radiator view so that teams can set up and action item radiator that switches through automatically.

Solves issue #40

### Demo
[GIF](https://media.giphy.com/media/9mTd19OIBa0ycwUUbj/giphy.gif)

![image](https://user-images.githubusercontent.com/6293337/44889081-6260f180-aca2-11e8-9b1a-6b36b78606ed.png)

## Testing Instructions
1. Click on the "Actions Radiator" button in the header
2. You should see the a page that looks similar to the screenshot above.
3. Click on the autoscroll button at the bottom of the page and make sure it autoscrolls through the page.
